src/middleware/extractAndBundle.js

```diff
@@ -32,7 +32,7 @@ function extractAndBundle (req, res) {
         dll: files[1]
       });
       verifyAvailability.isAvailable = true;
-
+      return
```

Comment:
Nitpick, I think a `;` should go after this, otherwise result of `exec` will be returned.